src/pages/CarList/CarList.tsx

```diff
@@ -74,12 +74,12 @@ const CarList = (): JSX.Element => {
 
   return (
     <div className="car-list-container">
-      <div className="car-list-tab-search">
-        <div className="car-list-btn-group">
+      <div className="car-list-tab-search-container">
+        <div className="car-list-tabs">
           <div>
             <button
               className={`${
-                selectedCarType === CarType.ALL_CARS ? "active-tab" : ""
+                selectedCarType === CarType.ALL_CARS ? "car-list-active-tab--btn" : ""
```

Comment:
--btn is not a modifier
the ideal class names would be
`car-list`
`car-list__container`
`car-list__tabs`
`car-list__tab--active`